src/components/loader-spinner/loader-spinner.pw.tsx

```diff
@@ -0,0 +1,286 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import {
+  loaderSpinnerWrapper,
+  loaderSpinnerSvg,
+  loaderSpinnerOuterArc,
+  loaderSpinnerInnerArc,
+  loaderSpinnerVisibleLabel,
+  loaderSpinnerHiddenLabel,
+} from "../../../playwright/components/loader-spinner/index";
+import { checkAccessibility } from "../../../playwright/support/helper";
+import LoaderSpinnerComponent from "./components.test-pw";
+import {
+  LOADER_SPINNER_SIZE_PARAMS,
+  LOADER_SPINNER_SIZES as sizes,
+  LOADER_SPINNER_VARIANTS as variants,
+} from "./loader-spinner.config";
+
+test.describe("Prop checks for Loader Spinner component", () => {
+  test("when the 'size' prop is passed as 'extra-small' the component wrapper has a flex-direction of row", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<LoaderSpinnerComponent size="extra-small" />);
+
+    await expect(loaderSpinnerWrapper(page)).toHaveCSS("flex-direction", "row");
+  });
+
+  (["small", "medium", "large", "extra-large"] as const).forEach(
+    (spinnerSizes) => {
+      test(`when the 'size' prop is passed as '${spinnerSizes}' the component wrapper has a flex-direction of column`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<LoaderSpinnerComponent size={spinnerSizes} />);
+
+        await expect(loaderSpinnerWrapper(page)).toHaveCSS(
+          "flex-direction",
+          "column"
+        );
+      });
+    }
+  );
+
+  sizes.forEach((spinnerSizes) => {
+    test(`when the 'size' prop is passed as '${spinnerSizes}' the svg circle wrapper has the correct dimensions`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<LoaderSpinnerComponent size={spinnerSizes} />);
+
+      await expect(loaderSpinnerSvg(page)).toHaveCSS(
+        "height",
+        `${LOADER_SPINNER_SIZE_PARAMS[spinnerSizes].wrapperDimensions}px`
+      );
+      await expect(loaderSpinnerSvg(page)).toHaveCSS(
+        "width",
+        `${LOADER_SPINNER_SIZE_PARAMS[spinnerSizes].wrapperDimensions}px`
+      );
+    });
+  });
```

Comment:
**suggestion(non-blocking)**: given we are asserting the `min-height` and `min-width` in another test, we probably don't need to assert the `height` and `width` as well. What do you think?

```suggestion
```